simplefs/simplefs.go

```diff
@@ -961,6 +980,42 @@ func (k *SimpleFS) SimpleFSCopy(ctx context.Context, arg keybase1.SimpleFSCopyAr
 		})
 }
 
+// SimpleFSSymlink starts making a symlink of a file or directory
+func (k *SimpleFS) SimpleFSSymlink(ctx context.Context, arg keybase1.SimpleFSSymlinkArg) (err error) {
+	// This is not async.
+	ctx, err = k.startSyncOp(ctx, "Symlink", arg)
+	if err != nil {
+		return err
+	}
+	defer func() { k.doneSyncOp(ctx, err) }()
+
+	// Get root FS of dst
+	t, tlfName, restOfDstPath, finalDstElem, err := remoteTlfAndPath(arg.Dest)
+	if err != nil {
+		return err
+	}
+	tlfHandle, err := libkbfs.GetHandleFromFolderNameAndType(
+		ctx, k.config.KBPKI(), k.config.MDOps(), tlfName, t)
+	if err != nil {
+		return err
+	}
+	fs, err := libfs.NewFS(
+		ctx, k.config, tlfHandle, libkbfs.MasterBranch, "", "",
+		keybase1.MDPriorityNormal)
+	if err != nil {
+		return err
+	}
+	srcRawPath, err := rawPathFromPath(arg.Src)
+	if err != nil {
+		return err
+	}
+
+	err = fs.Symlink(
+		stdpath.Clean(srcRawPath),
+		stdpath.Join(restOfDstPath, finalDstElem))
```

Comment:
If you use `getFS()` above, you won't need to do this join -- `finalDstElem` would be enough.